packages/flutter/lib/src/scheduler/binding.dart

```diff
@@ -51,9 +52,17 @@ typedef void FrameCallback(Duration timeStamp);
 typedef bool SchedulingStrategy({ int priority, SchedulerBinding scheduler });
 
 class _TaskEntry {
-  const _TaskEntry(this.task, this.priority);
+  _TaskEntry(this.task, this.priority) {
+    assert(() {
+      debugStack = StackTrace.current;
+      return true;
+    }());
+  }
   final VoidCallback task;
   final int priority;
+
+  // debug-mode field
```

Comment:
I'd remove this comment.  That's what the "debug" prefix on the name means already.

Reply:
done